### PR TITLE
WDU 24 user identification implementation

### PR DESCRIPTION
**Pull request by nlasagni**
Implemented Login component of the mobile app for the WeDriveU system.

### CODE REVIEW

File: mobile/src/main/java/com/wedriveu/mobile/login/viewmodel/LoginViewModelImpl.java [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    }
+
+    private void setViewId(String viewId) {
+        mViewId = viewId;
+    }
+
+    /*
+    The deprecated method has been chosen because the minTargetVersion is 19.
+    Switch to >=23 in order to use the new onAttach method.
+*/
```
Comment (mbaldassarri) [inline mobile/src/main/java/com/wedriveu/mobile/login/viewmodel/LoginViewModelImpl.java]: please have a look at the comment indentation

File: mobile/src/main/java/com/wedriveu/mobile/service/login/LoginService.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
+package com.wedriveu.mobile.service.login;
+
+/**
+ *
+ * This service interacts with the AuthenticationService backend to perform the user login.
+ * This service is part of the Model in MVVM pattern.
+ *
+ * @Author Marco Baldassarri
+ * @Since 12/07/2017
```
Comment (mbaldassarri) [inline mobile/src/main/java/com/wedriveu/mobile/service/login/LoginService.java]: have you checked the javadoc syntax?

File: mobile/src/main/res/layout/fragment_login_view.xml [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_below="@+id/linearLayout"
+            android:layout_centerHorizontal="true"
+            android:layout_marginTop="68dp"
+            android:id="@+id/loginButton" />
+
+    </LinearLayout>
+
+
```
Comment (mbaldassarri) [inline mobile/src/main/res/layout/fragment_login_view.xml]: remove spaces here

File: mobile/src/main/java/com/wedriveu/mobile/login/viewmodel/LoginViewModelImpl.java (hunk 1 above)
Comment (nlasagni) [inline mobile/src/main/java/com/wedriveu/mobile/login/viewmodel/LoginViewModelImpl.java]: OK

File: mobile/src/main/java/com/wedriveu/mobile/service/login/LoginService.java (hunk 2 above)
Comment (nlasagni) [inline mobile/src/main/java/com/wedriveu/mobile/service/login/LoginService.java]: I'll check.

File: mobile/src/main/res/layout/fragment_login_view.xml (hunk 3 above)
Comment (nlasagni) [inline mobile/src/main/res/layout/fragment_login_view.xml]: OK

**Pull request merged**